File: Haskell/beginner/1071.hs
-- https://judge.beecrowd.com/pt/problems/view/1071
-- Soma de Impares Consecutivos I

-- Main body
main :: IO ()
main = do
    -- Scanning values of interval
    usr_value1 <- readLn :: IO Int
    usr_value2 <- readLn :: IO Int

    -- Adjusting interval
    let (start, end) = if usr_value1 > usr_value2 
        then (usr_value2, usr_value1) 
        else (usr_value1, usr_value2)
    
    -- Calculating sum
    let sumOdd = sum [x | x <- [start+1..end-1], odd x]

    -- Showing results
    print sumOdd

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1071.c
// https://judge.beecrowd.com/pt/problems/view/1071
// Soma de Impares Consecutivos I

// Libraries
#include <stdio.h>

// Main Function
int main() {
    // Starting code and declarations
    int usr_input1, usr_input2;
    int temp_storage, sum;

    // Scanning user input
    scanf("%d", &usr_input1);
    scanf("%d", &usr_input2);

    // Adjusting interval
    if (usr_input1 > usr_input2) {
        temp_storage = usr_input1;
        usr_input1 = usr_input2;
        usr_input2 = temp_storage;
    }

    // Starting sum dump
    sum = 0;    

    // Iterating
    for (int i = usr_input1 + 1; i < usr_input2; i++) {
        if (i % 2 != 0){
            sum += i;
        }        
    }
    
    // Showing results
    printf("%d\n", sum);

    // Ending code
    return 0;
}

// By Fernando Serra
// https://github.com/fernandosserra